src/components/BillCard/BillCard.svelte

```diff
@@ -1,9 +1,16 @@
 <script lang="ts">
 	import { ArrowRight } from 'carbon-icons-svelte';
 	import BillStatusTag from '$components/BillStatusTag/BillStatusTag.svelte';
-	import { type Bill } from '$models/bill';
+	import { BillProposerType, type Bill } from '$models/bill';
 	import { twMerge } from 'tailwind-merge';
-	import Proposer from '$components/Proposer/Proposer.svelte';
+	import Proposer, {
+		type AssemblyProposer,
+		type PeopleProposer,
+		type PoliticianProposer
+	} from '$components/Proposer/Proposer.svelte';
+	import { getRoleHistoryAtTime } from '$models/politician';
+	import { AssemblyName } from '$models/assembly';
```

Comment:
There are some unused imports here, maybe it came from auto import while you were refactoring the code. This should be my last point, I'm so happy with the rest of your work! 🙌